rocketrus.go

```diff
@@ -198,10 +198,10 @@ func (rh *RocketrusHook) Fire(e *logrus.Entry) error {
 	}
 
 	msg := &models.Attachment{
-		Color: color,
-		Title: e.Level.String() + " log",
-		Ts:    e.Time.String(),
-		Text:  e.Message,
+		Color:     color,
+		Title:     e.Level.String() + " log",
+		Timestamp: e.Time.String(),
```

Comment:
where is the doc?
I didn't see any change.
https://github.com/RocketChat/Rocket.Chat.Go.SDK/blob/master/models/message.go#L47
https://rocket.chat/docs/developer-guides/rest-api/chat/postmessage/#attachments-detail

Reply:
you changed the field Ts in the SDK https://github.com/RocketChat/Rocket.Chat.Go.SDK/commit/2a98370dd0f119b89235fb7bcdec2a4f748ca4a4#diff-c2de9728f932398a66758c56ed794296  to Timestamp but not in rocketrus.go